xcb.el

```diff
@@ -465,10 +467,12 @@ classes of EVENT (since they have the same event number)."
             (plist-put plist key
                        (if (child-of-class-p event 'xcb:-generic-event)
                            'xge 'xkb))))
-    (setq listeners (lax-plist-get plist event-number))
+    (setq listeners (plist-get plist event-number #'equal))
     (setf (slot-value obj 'event-plist)
-          (lax-plist-put plist event-number (append listeners
-                                                    (list listener))))))
+          (plist-put plist
+                     event-number
+                     (append listeners (list listener))
+                     #'equal))))
```

Comment:
We cannot yet use plist-get/put with a predicate argument since this argument has only been introduced in Emacs 29. At the same time lax-plist-get/put was marked as deprecated.

Our options:

1. We live with the deprecation message.
2. We define our own macro/alias, handling the compatibility issue.
3. We start to depend on Compat and use (compat-call plist-get list value 'equal)

Reply:
Ah... That's annoying. I think the best option is to live with the errors for now.

Reply:
I've reverted that change for now.

Reply:
In the longer term I'd opt for Compat. It is supposed to handle exactly such issues and I don't want to do that repeatedly. Alternatively we could also opt for a more aggressive strategy requiring newer Emacs versions. Compat mostly solved compatibility issues for many of my packages, however this does not work for packages which touch deeper Emacs internals or may be affected by UI bug fixes and improvements (e.g. tab bar changes, see https://github.com/emacs-exwm/exwm/issues/8#issuecomment-1907383947).